nifi-docker/dockerhub/pom.xml

```diff
@@ -61,7 +61,7 @@
                                 <configuration>
                                     <arguments>
                                         <argument>${project.version}-dockerhub</argument>
-                                        <argument>1.6.0</argument>
+                                        <argument>1.7.0</argument>
```

Comment:
Should be 1.8.0 now.

Reply:
unfortunately it cannot, since there is no released binary for 1.8 yet